typegate/tests/runtimes/prisma/full_prisma_mapping.py

```diff
@@ -79,4 +80,38 @@ def prisma(g: Graph):
                 "where": {"id": 10007},
             }
         ),
+        testExecuteRaw=db.execute(
+            'UPDATE "Post" SET title = ${replacement} WHERE title LIKE ${title}',
+            t.struct(
+                {
+                    "title": t.string().set("%Title 2%"),
+                    "replacement": t.string(),
+                }
+            ),
+            EffectUpdate(True),
+        ),
+        testQueryRaw=db.query_raw(
+            """
+                WITH tmp AS (
+                    SELECT id, title, (views + likes) as reactions FROM "Post"
+                    WHERE title LIKE ${title}
+                ) SELECT * FROM tmp WHERE id IN (${one}, ${two})
```

Comment:
Great! I believe there is one more use case with the `select * from a WHERE b IN (${list})` where `t.array(t.string())`. At the list cardinality could be varying we need to support any size.